src/uct/ib/base/ib_md.c

```diff
@@ -544,8 +544,7 @@ ucs_status_t uct_ib_dereg_mr(struct ibv_mr *mr)
         return UCS_OK;
     }
 
-    ucs_trace("ibv_dereg_mr(mr=%p addr=%p length=%zu)", mr, mr->addr,
-              mr->length);
+    ucs_trace("ibv_dereg_mr(mr=%p length=%zu)", mr, mr->length);
```

Comment:
why removed addr? even if NULL it's ok to print.

Reply:
Reverted.